ddl/models.py

```diff
@@ -47,20 +55,43 @@ class Meta:
 
   name = models.TextField()
   menu = models.ForeignKey(Menu, blank=True)
+  address = models.ForeignKey(Address)
+  delivery_fee = models.IntegerField()
 
 class OrderCart(models.Model):
   class Meta:
     db_table = 'order_cart'
 
+  #future fields: promo code, etc
   order_items = models.ManyToManyField(OrderItem, blank=True)
 
+
+  #Add food items to the order cart
+  #sketch???
+  def update(self, order_item_list):
```

Comment:
checkout this: https://github.com/doordash/doordash-backend-standards/blob/master/business-logic.md#crud-organization